### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by srittau**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation-issue.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Documentation issue
+about: Report a problem or suggest changes for the documentation at https://typing.readthedocs.io/
+title: ''
```
Comment (sobolevn) [inline .github/ISSUE_TEMPLATE/documentation-issue.md:4]: There's no need to add `title: ''`

**Review by sobolevn — COMMENTED** [review]
I also think that we can add links to mypy / pyright issue trackers.

It looks something like this: 
Снимок экрана 2021-11-04 в 18 08 35

Definition: https://github.com/dry-python/returns/blob/master/.github/ISSUE_TEMPLATE/config.yml

File: .github/ISSUE_TEMPLATE/typing-extensions-issue.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: typing-extensions issue
+about: Report a problem or suggest changes for the typing-extension library
```
Comment (JelleZijlstra) [inline .github/ISSUE_TEMPLATE/typing-extensions-issue.md:3]:
```suggestion
about: Report a problem or suggest changes for the typing-extensions library
```

File: .github/ISSUE_TEMPLATE/documentation-issue.md (hunk 1 above)
Comment (srittau) [reply .github/ISSUE_TEMPLATE/documentation-issue.md:4]: This was added automatically by the templates generator.

**Comment by srittau** [comment]
I went ahead and merged this. We can always improve this later or revert if necessary.

**Pull request merged**